test/cri-containerd/test-images/nanoserver-gracefultermination-repro/latest/delayed-shutdown.go

```diff
@@ -42,4 +42,4 @@ func main() {
 	}
 
 	fmt.Println("Goodbye")
-}
\ No newline at end of file
+}
```

Comment:
This empty line was actually added automatically by VS code formatter. Since it's consistent with other go files, I'll leave it. Same with the empty line in  ./test-images/servercore-gracefultermination-repro/latest/delayed-shutdown.go